qiskit_experiments/framework/restless_mixin.py

```diff
@@ -128,20 +140,54 @@ def enable_restless(
                 f"a smaller repetition delay for the restless experiment."
             )
 
-    def _get_restless_processor(self) -> DataProcessor:
+    def _get_restless_processor(
+        self, meas_level: int = 2, meas_return: str = "average"
```

Comment:
`meas_level` makes sense here, but I don't think that `meas_return` does since we always need each shot.

Reply:
Thanks, I removed `meas_return` here: [315b93a](https://github.com/Qiskit/qiskit-experiments/pull/752/commits/315b93a02555acc1352497c0edb0994d63c419e2)